usaon_benefit_tool/templates/macros/javascript_highcharts.j2

```diff
@@ -17,13 +27,19 @@
       series: [{
         keys: ['from', 'to', 'weight'],
         data: {{ sankey_series | safe }},
+        dataLabels: {
+          enabled:true,
+          style: {
+            fontSize:'17px'
+          }
+        }
```

Comment:
This was making the font size a bit too large on each node, I think it may get a bit too cluttered on bigger charts. Temporarily removed this, but we may bring it back later :)